coretex/cli/modules/node.py

```diff
@@ -0,0 +1,65 @@
+from typing import Any, Dict
+
+import logging
+
+from . import docker
+
+
+DOCKER_CONTAINER_NAME = "coretex_node"
+DOCKER_CONTAINER_NETWORK = "coretex_node"
+
+
+class NodeException(Exception):
+    pass
+
+
+def pull(repository: str, tag: str) -> None:
+    try:
+        docker.imagePull(f"{repository}:{tag}")
+    except BaseException as ex:
+        logging.getLogger("cli").debug(ex, exc_info = ex)
+        raise NodeException("Failed to fetch latest node version")
+
+
+def start(dockerImage: str, config: Dict[str, Any]) -> None:
+    dockerImage = f"coretexai/coretex-node:latest-{config['image']}"
```

Comment:
`dockerImage` is passed in as a parameter but it is getting redefined here, is this intended behaviour?